build.sbt

```diff
@@ -105,7 +105,11 @@ lazy val testing = libraryProject("testing")
     description := "Instances and laws for testing http4s code",
     libraryDependencies ++= Seq(
       specs2Matcher,
+      munitCatsEffect,
+      munitDiscipline
     ),
+    unusedCompileDependenciesFilter -= moduleFilter(organization = "org.typelevel", name = "discipline-munit"),
```

Comment:
I need this to pass CI but I'm not sure why, we are using classes from both libs